index.ts

```diff
@@ -18,7 +18,7 @@ export { Card } from './components/Card';
 export { Checkbox } from './components/Checkbox';
 // export { Code } from './components/Code';
 export { Container } from './components/Container';
-export { Elevation } from './components/Elevation';
+export { Elevation, elevationVariant } from './components/Elevation';
```

Comment:
WDYT of giving a more meaningful name for this var?

```suggestion
export { Elevation, elevationVariant as elevationStyles } from './components/Elevation';
```

Reply:
after discussion, @paulocfjunior proposed `elevationVariants`